manuscript/index.qmd

```diff
@@ -149,7 +175,7 @@ Say if it looked okay and reference SI
 - We do not explore more complex prior models such as splines and gaussian processes
 - We focus our efforts on situational awareness and hence real-time performance. This means we do not focus on retrospective performance which may have different characteristics.
 - We did not perform full simulation-based calibration.
-- Our simulations are produced by a model that is similar to the renewal process inference method and so represents a "best" case for this method. Potential future work could explore other versions of the infection generation process backing the simulations but we feel this choice makes sense given that the renewal process best reflects our mechanistic understanding of how transmission works of the models we explore here.
+- Our simulations are produced by a model that is similar to the renewal process inference method and so represents a "best" case for this method. It would be potentially be stronger to simulate from an SEIR process directly.
```

Comment:
```suggestion
- Forward simulations of infections are generated from a renewal process with a time-varying reproduction number $\mathcal{R}_t$ and next generation time distribution $w(t)$ **CITATIONS**. This is a flexible approach to modelling the underlying infection dynamics that absorbs other popular modelling choices as special cases, such as the the SEIR model **CITE DIECKMANN AND HEESTERBEEK's book 2000 OR OTHER OPTIONS**.

Our simulations are produced by a model that is similar to the renewal process inference method and so represents a "best" case for this method. It would be potentially be stronger to simulate from an SEIR process directly.
```